### PR TITLE
Remove tags from a few odds and ends topics

### PR DESCRIPTION
**Pull request by hamishwillee**
Removes tags from topics in groups that were dealt with already or that are not covered by other groups of interest.

Part of https://github.com/mdn/mdn/issues/262

### CODE REVIEW

File: files/en-us/web/text_fragments/index.md [hunk 1]
```diff
@@ -1,19 +1,6 @@
 ---
 title: Text fragments
 slug: Web/Text_fragments
```
Comment (hamishwillee) [inline files/en-us/web/text_fragments/index.md:3]: Note, no page type, should we add landing page?

File: files/en-us/web/tutorials/index.md [hunk 2]
```diff
@@ -1,20 +1,6 @@
 ---
 title: Tutorials
 slug: Web/Tutorials
```
Comment (hamishwillee) [inline files/en-us/web/tutorials/index.md:3]: Note, no page type - is there one we can/should use?

File: files/en-us/web/opensearch/index.md [hunk 3]
```diff
@@ -1,14 +1,6 @@
 ---
 title: OpenSearch description format
 slug: Web/OpenSearch
```
Comment (hamishwillee) [inline files/en-us/web/opensearch/index.md:3]: Is there a page type we can use for this? Note, I don't think it should stop us merging as there are no "status" tags.

Comment (teoli2003) [reply files/en-us/web/opensearch/index.md:3]: I would put "guide" or "landing-page"

File: files/en-us/web/text_fragments/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/text_fragments/index.md:3]: Here "landing-page"

(I will not block the merge here)

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**